extensions/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB/AzureCosmosDBVectorSearchType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// ReSharper disable InconsistentNaming
+using Newtonsoft.Json;
```

Comment:
ditto, I would avoid introducing Newtonsoft in the project, and use .NET default JSON serializer instead